src/core/inductive.ml

```diff
@@ -0,0 +1,63 @@
+(** Builtin configuration for induction. *)
+
+open Timed
+open Pos
+open Console
+open Terms
+open Print
+
+type config =
+  { symb_Prop : sym (** : TYPE. Type of propositions. *)
+  ; symb_prf  : sym (** : Prop → TYPE.
+                        Interpretation of propositions as types. *) }
+
+(** [get_config ss pos] build the configuration using [ss]. *)
+let get_config : Sig_state.t -> Pos.popt -> config = fun ss pos ->
+  let builtin = Builtin.get ss pos in
+  { symb_Prop = builtin "Prop"
+  ; symb_prf  = builtin "π" }
```

Comment:
Please use `"P"` for the moment (as for the rewrite and why3 tactics), we will change this later.